packages/extensions/venia-sample-payments-checkmo/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@magento/venia-sample-payments-checkmo",
+  "version": "0.0.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Provides demo Check Money Order Payment Method for PWA Studio.",
+  "main": "./intercept.js",
+  "scripts": {
+    "clean": " "
+  },
+  "repository": "github:magento/pwa-studio",
+  "license": "(OSL-3.0 OR AFL-3.0)",
+  "peerDependencies": {
+    "@magento/peregrine": ">= 8.0.0",
+    "@magento/pwa-buildpack": "~8.0.1",
+    "@magento/venia-ui": ">= 5.0.0",
```

Comment:
@larsroettig   With our current capabilities this `>=` would cause manual check for every package publishing in future. Can these dependencies be in sync with rest of the project with latest versions? something like - 
```
    "@magento/peregrine": "~9.0.0",
    "@magento/pwa-buildpack": "~8.0.1",
    "@magento/venia-ui": "~6.0.1",
```

Reply:
@dpatil-magento Sure i can do but this mean before publish you replace any version ?

Reply:
@larsroettig Yes, we update dependencies to respective release latest versions.  Example - When we go for next release and `peregrine` we identify as `10.0.0` then in above case  `"@magento/peregrine": "~9.0.0",` will be replaced to `"@magento/peregrine": "~10.0.0",`

Reply:
Good to know thanks pull commit the chnage now

Reply:
https://github.com/magento/pwa-studio/pull/2969/commits/07f0164da326b1df82bf1ca1dac211d59933936d done

Reply:
thanks @larsroettig